source/evidencevariable/structuredefinition-EvidenceVariable.xml

```diff
@@ -568,5 +568,207 @@
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="positiveInt"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.code">
+			<path value="EvidenceVariable.constraint.code"/>
+			<short value="Rule for acceptable data values"/>
+			<definition value="A rule, such as a format or other expectation, for the data values."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.expression">
+			<path value="EvidenceVariable.constraint.expression"/>
+			<short value="Rule for acceptable data values, as an Expression"/>
+			<definition value="A rule, such as a format or other expectation, for the data values, expressed as an Expression."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="Expression"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.expectedValueSet">
+			<path value="EvidenceVariable.constraint.expectedValueSet"/>
+			<short value="List of anticipated values used to express value of the variable"/>
+			<definition value="List of anticipated values used to express value of the variable, used with variables that have a codeable concept-based datatype."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="Reference"/>
+				<targetProfile value="http://hl7.org/fhir/StructureDefinition/ValueSet"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.expectedUnitsValueSet">
+			<path value="EvidenceVariable.constraint.expectedUnitsValueSet"/>
+			<short value="List of anticipated values used to express units for the value of the variable"/>
+			<definition value="List of anticipated values used to express units for the value of the variable, used with variables that have a Quantity-based datatype."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="Reference"/>
+				<targetProfile value="http://hl7.org/fhir/StructureDefinition/ValueSet"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.anyValueAllowed">
+			<path value="EvidenceVariable.constraint.anyValueAllowed"/>
+			<short value="Permissibility of unanticipated value used to express value of the variable"/>
+			<definition value="Whether the value expressed for a variable is allowed to not be restricted to the expected value set."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="boolean"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.missingDataMeaning">
+			<path value="EvidenceVariable.missingDataMeaning"/>
+			<short value="How missing data can be interpreted"/>
+			<definition value="A method or transformation applied for missing data."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
```

Comment:
The ticket indicates missingDataMeaning should have an example value set?

Reply:
Added the binding.